backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -421,6 +426,8 @@ public Response enableEndpoint(@Context SecurityContext sec, @PathParam("id") UU
     @DELETE
     @Path("/{id}/enable")
     @RolesAllowed(ConsoleIdentityProvider.RBAC_WRITE_INTEGRATIONS_ENDPOINTS)
+    @Operation(summary = "Disable an endpoint", description = "Disables an endpoint so that the endpoint will not be executed after an operation that uses the endpoint is restarted. An operation must be restarted to disable the endpoint in that operation. Disable an endpoint when you want to stop it from running and might want to re-enable it in the future.")
```

Comment:
Similar as above

```suggestion
    @Operation(summary = "Disable an endpoint", description = "Disables an endpoint so that the endpoint will not be executed after an operation that uses the endpoint is started. An operation that is already running could still execute the endpoint. Disable an endpoint when you want to stop it from running and might want to re-enable it in the future.")
```